Automattic/wp-calypso
# VideoPress User intent: Add design and survey to the "Other" Intent

Related to # https://github.com/Automattic/greenhouse/issues/1841

## Proposed Changes

* add a new modal for the "Other" step
* the user text response should be sent to a crowdsignal "project"
* should match the figma designs VWoFEOpEaOyeLspB9GoRvm-fi-3092_13890

| question | thank you |
| - | - |
| <img width="832" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4081020/52e09bd9-a28d-4442-b3a4-2a1281e80a77"> | <img width="917" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4081020/4a441bb1-4872-4ab7-950a-b40a75e64e94"> |

## Testing Instructions



* `yarn && yarn start`
* `/setup/videopress`
* select the "Other" item
* it should match the figma design
* fill out the question and submit
* you should see the "Thank you" message
* close and reopen the "Other" item
* You should see the survey again... this was intentional since someone may have more to add?

## Pre-merge Checklist
